pkgs/db/utils.go

```diff
@@ -0,0 +1,266 @@
+package db
```

Comment:
This file should be called "stub.go". Please decompose all stub implementation from the pure logic of the service